ppl/zqd/search/search.go

```diff
@@ -86,6 +86,7 @@ func (s *SearchOp) Run(ctx context.Context, store storage.Storage, output Output
 	switch st := store.(type) {
 	case *archivestore.Storage:
 		return driver.MultiRun(ctx, d, s.query.Proc, zctx, st.MultiSource(), driver.MultiConfig{
+			Distributed: distributed,
```

Comment:
How about this instead of adding a parameter?

```suggestion
			Distributed: parallelism > 0,
```

Reply:
That works. There's a knot here that needs to be untangled.

Reply:
You could conceivably pass in parallelism > 0 intending to run on a single host with multiple scanners, but we aren't currently doing this.

Reply:
Right. I vote for crossing that bridge when we come to it.

By the way, this breakage is all my fault because I left this out of #2178.

Reply:
I'm aware 😄